subhub/db.py

```diff
@@ -70,7 +70,7 @@ def new_user(
 
     def get_user(self, uid: str) -> Optional[SubHubAccountModel]:
         try:
-            subscription_user = self.model.get(uid, consistent_read=True)
+            subscription_user: Any = self.model.get(uid, consistent_read=True)
```

Comment:
what is the benefit of typing to `Any`?

Reply:
It reads to me as this should respond with a `SubHubAccountModel`.

Reply:
@scottidler @bowlofstew Changing Any to SubHubAccountModel causes the following error: `error: Incompatible types in assignment (expression has type "subhub.db.SubHubAccountModel@48", variable has type "subhub.db.SubHubAccountModel")`

Reply:
Yeah, like I say in the comment above, I think it might have something to do with that class being defined twice, which has never sat well with me anyway.  I could be wrong though, but we should figure it out.